### PR TITLE
0.7.0

### PR DESCRIPTION
**Pull request by ashleygwilliams**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,182 @@
 # Changelog
 
+## 🌤️ 0.7.0
+
+- ### ✨ Features
+
+  - **Support passing arbitrary arguments to `cargo test` via `wasm-pack test` - [chinedufn], [issue/525] [pull/530]**
+
+    `wasm-pack test` is an awesome command that wraps `cargo test` in a way that helps provide you some nice out of the
+    box configuration and setup. However, you may find yourself wanting to leverage the full funcationality of `cargo test`
```
Comment (drager) [inline CHANGELOG.md]:
```suggestion
    box configuration and setup. However, you may find yourself wanting to leverage the full functionality of `cargo test`
```

File: Cargo.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "wasm-pack"
 description = "📦✨ your favorite rust -> wasm workflow tool!"
-version = "0.6.0"
+version = "0.6.1"
```
Comment (drager) [inline Cargo.toml]:
```suggestion
version = "0.7.0"
```

File: docs/index.html [hunk 3]
```diff
@@ -42,9 +42,9 @@ <h1><code>wasm-pack</code></h1>
           <h2>📦✨ your favorite rust -> wasm workflow tool!</h2>
         </div>
         <div class="five columns" id="installer">
-          <a class="button button-primary" href="/wasm-pack/installer">✨ Install wasm-pack 0.6.0 ✨</a>
-          <p>15 Jan 2019 |
-            <a href="https://github.com/rustwasm/wasm-pack/releases/tag/v0.6.0">
+          <a class="button button-primary" href="/wasm-pack/installer">✨ Install wasm-pack 0.6.1 ✨</a>
+          <p>16 Jan 2019 |
+            <a href="https://github.com/rustwasm/wasm-pack/releases/tag/v0.6.1">
```
Comment (drager) [inline docs/index.html]:
```suggestion
            <a href="https://github.com/rustwasm/wasm-pack/releases/tag/v0.7.0">
```

File: docs/index.html [hunk 4]
```diff
@@ -42,9 +42,9 @@ <h1><code>wasm-pack</code></h1>
           <h2>📦✨ your favorite rust -> wasm workflow tool!</h2>
         </div>
         <div class="five columns" id="installer">
-          <a class="button button-primary" href="/wasm-pack/installer">✨ Install wasm-pack 0.6.0 ✨</a>
-          <p>15 Jan 2019 |
-            <a href="https://github.com/rustwasm/wasm-pack/releases/tag/v0.6.0">
+          <a class="button button-primary" href="/wasm-pack/installer">✨ Install wasm-pack 0.6.1 ✨</a>
```
Comment (drager) [inline docs/index.html]:
```suggestion
          <a class="button button-primary" href="/wasm-pack/installer">✨ Install wasm-pack 0.7.0 ✨</a>
```

File: docs/index.html [hunk 5]
```diff
@@ -42,9 +42,9 @@ <h1><code>wasm-pack</code></h1>
...
+          <p>16 Jan 2019 |
```
Comment (drager) [inline docs/index.html]: Be sure to update this when the release happens :blush:

Comment (ashleygwilliams) [reply docs/index.html]: Yes! Good call!

**Comment by ashleygwilliams** [comment]
ok! waiting on CI for #577 then will merge and rebase. changelog should be ready to go. @fitzgen @alexcrichton wanna give this a look?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,299 @@
...
+
+    [file an issue]: https://github.com/rustwasm/wasm-pack/issues/new/choose
+    [pull/547]: https://github.com/rustwasm/wasm-pack/pull/547
+    [issue/298]: https://github.com/rustwasm/wasm-pack/issues/298
+
+  - **Add support for `--target web` - [alexcrichton], [pull/567]**
+
+    Recently, `wasm-bindgen` add a new target- `web`. This new target is similar to the `no-modules` target, in that
+    it is designed to generate code that should be loaded directly in a browser, without the need of a bundler. As 
+    opposed to the `no-modules` target, which produces and IIFE (Immediately Invoked Function Expression), this target
```
Comment (fitzgen) [inline CHANGELOG.md]:
```suggestion
    opposed to the `no-modules` target, which produces an IIFE (Immediately Invoked Function Expression), this target
```

typo :)

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,299 @@
...
+    Previously, when view `--help`, the command description for `login` showed:
+    `👤  Add a registry user account!` This could be confusing for folks, so now it's been updated to read:
+    `👤  Add an npm registry user account!`, which is much clearer!
+
+    [killercup]: https://github.com/killercup
+    [pull/539]: https://github.com/rustwasm/wasm-pack/pull/539
+
+  - **Wasm is a contraction, not an acronym - [fitzgen], [pull/555]**
+
+    Ever wond how you're *actually* supposed to refer to WebAssembly in short-form? WASM? wasm? For the pedants
```
Comment (fitzgen) [inline CHANGELOG.md]:
```suggestion
    Ever wonder how you're *actually* supposed to refer to WebAssembly in short-form? WASM? wasm? For the pedants
```

pedanting on the pedantry &mdash; x to the z

Comment (fitzgen) [reply CHANGELOG.md]: (yes, this is an xzibit reference)

**Pull request merged**